api/pkg/types/answers.go

```diff
@@ -151,3 +152,22 @@ func (a BoolAnswer) Value() (driver.Value, error) {
 func (a *BoolAnswer) Validate(q Question) error {
 	return nil
 }
+
+type EmailAnswer struct {
+	AnswerValue string `json:"value"`
+}
+
+func (a EmailAnswer) Value() (driver.Value, error) {
+	return json.Marshal(a)
+}
+
+func (a *EmailAnswer) Validate(q Question) error {
+	re := regexp.MustCompile(`^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$`)
```

Comment:
This is not ideal email validation. Let's use some package like ozzo/validation

Reply:
have added ozzo/validations to email let me know if you want me to add this to other answer type validators as well.